Formula/ansible.rb

```diff
@@ -5,6 +5,7 @@ class Ansible < Formula
   homepage "https://www.ansible.com/"
   url "https://releases.ansible.com/ansible/ansible-2.3.0.0.tar.gz"
   sha256 "299f3907cd566a20e163942fa82b6afc86ef89c2726ba503b90c1a651e82a458"
+  revision 2
```

Comment:
@ilovezfs I'm not sure how to clear this change request at this point. It's now out of date (as previously mentioned) but it's showing up as a red block on the pull request.